contacts.module

```diff
@@ -85,7 +86,40 @@ function template_preprocess_contacts_dash_tabs(array &$variables) {
  *   - region_attributes: HTML markup attributes for the content wrapper.
  */
 function template_preprocess_contacts_dash_tab_content(array &$variables) {
+  $variables['messages'] = $variables['content']['messages'];
   $variables['region_attributes'] = new Attribute($variables['region_attributes']);
+  /* @var \Drupal\contacts\Entity\ContactTab $tab */
+  $tab = \Drupal::service('contacts.tab_manager')->getTabByPath($variables['user'], $variables['subpage']);
+  $layout = $tab->get('layout') ?: 'contacts_tab_content.stacked';
+
+  // @todo switch to core layout manager plugin.manager.core.layout?
```

Comment:
Yes - definitely want to use the core layout module, including upping our core version requirement. No point implement the non core stuff...